src/descriptor.js

```diff
@@ -1933,6 +1933,15 @@ function createMessage(
     pbIdentifier
 ) {
     const members = [];
+    messageDescriptor.field = messageDescriptor.field.map(f => {
```

Comment:
IMHO - this shouldn't be the default behavior, but rather an parameter to the generator to enable the behavior of camelCasing.